packages/confirm-dialog/src/vaadin-confirm-dialog.js

```diff
@@ -308,8 +309,11 @@ class ConfirmDialog extends ElementMixin(ThemePropertyMixin(ControllerMixin(Poly
     this.addController(this._headerController);
 
     this._messageController = new SlotController(this, '', 'div', {
+      // Do not remove the default message node
```

Comment:
This comment sounds a bit unclear if you aren't familiar with this specific issue. I would suggest rephrasing it:

```suggestion
      // Allow providing multiple custom nodes in the default slot
```

```suggestion
      // Allow placing multiple custom nodes in the default slot
```

```suggestion
      // Allow the user to place multiple custom nodes in the default slot.
```